lib/reverse_words.rb

```diff
@@ -38,3 +38,7 @@ def reverse_words(my_words)
   end
   return my_words
 end
+
```

Comment:
Minor: since you're manipulating the string in place, you don't need to return my_words on line 39. Line 39 could simply be `return`.